string node_name
---
uint8 current_state
